migrations/message_store/00008_updatePrimaryKey_add_col.up.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE message RENAME TO message_backup;
+
+CREATE TABLE IF NOT EXISTS message (
+  pubsubTopic BLOB NOT NULL,
+  contentTopic BLOB NOT NULL,
+  payload BLOB,
+  version INTEGER NOT NULL,
+  timestamp INTEGER NOT NULL,
+  id BLOB,
+  messageHash BLOB, -- Newly added, this will be populated with a counter value
+  storedAt INTEGER NOT NULL,
+  CONSTRAINT messageIndex PRIMARY KEY (storedAt, messageHash)
```

Comment:
Have noted elsewhere, but:
1. I would imagine we only need `messageHash` as the primary key?
2. Not sure if we need to maintain the previous index on `(storedAt, id, pubsubTopic)` for query performance?

Reply:
1. Answered elsewhere
2. for query performance input from @Ivansete-status ?

Reply:
> ah yes! You are right @jm-clius, the `messageHash` will be the primary key

in that case, I will open a new PR since in this we also need to depend on amended digest (which I have already made locally in a separate PR against issue #2215 ) so it is better to open a new PR for the removal of `storedAt` from the primary key